test/C/src/zephyr/threaded/UserThreads.lf

```diff
@@ -5,7 +5,7 @@ target C {
     name: Zephyr,
     user-threads: 3
   },
-  threading: true,
+  single-threaded: false,
```

Comment:
Should this instead be a `workers` property?

Reply:
I would just remove the property and use the default number of workers.

Reply:
Same as the other one - I'll remove the property.